sos/plugins/kubernetes.py

```diff
@@ -65,6 +74,13 @@ def setup(self):
             kube_cmd = "kubectl "
             if path.exists('/etc/origin/master/admin.kubeconfig'):
                 kube_cmd += "--config=/etc/origin/master/admin.kubeconfig"
+            else:
+                if self.get_option('kubeconfig'):
+                    kubeconfig_path = self.get_option('kubeconfig')
+                else:
+                    kubeconfig_path = expanduser("~/.kube/config")
+                if kubeconfig_path != "" and path.exists(kubeconfig_path):
+                    kube_cmd += "--kubeconfig=" + kubeconfig_path + " "
```

Comment:
None of these lines will run if `icp=True`... how come they are being added here?

Reply:
Hi @bmr-cymru 
Pushed some update on the patch including the changes you requested. Can you check please??